_posts/2020-04-16-foodsearchinc.md

```diff
@@ -0,0 +1,50 @@
+---
+author: John
+title: foodsearchinc.com connects customers & restaurants during pandemic
+---
+A co-winner of our week four competition, [foodsearchinc.com](https://foodsearchinc.com/), hopes to connect people to comprehensive information about the food they desire, and how to easily locate it. 
```

Comment:
Add blank line just above here

Reply:
sorry where is here-this thing needs a highlighter of some sort...I'll take a quick look and see if it's obvious in the code.

Reply:
Generally is the highlighted (green) code showing me what I have, or what you'd like?